synapse/storage/databases/main/events_worker.py

```diff
@@ -600,6 +603,8 @@ async def get_events_from_cache_or_db(
             allow_rejected: Whether to include rejected events. If False,
                 rejected events are omitted from the response.
 
+            get_prev_content: whether to fetch previous event content
+
```

Comment:
I was a bit thrown by `previous event content`: I thought it referred to `prev_events`.
Maybe something like this?

> get_prev_content: add `prev_content` and `prev_sender` keys to the `unsigned` dict when fetching redaction events. Has no effect on non-redaction events.

Reply:
Sounds good to me!